app/views/mailbot/booth_rejection_mail.text.erb

```diff
@@ -0,0 +1,7 @@
+Dear <%= @booth.submitter.name %>,
```

Comment:
Why do we need these templates if organizers can create their own email templates that will be sent?